incubator/jaeger/templates/agent-svc.yaml

```diff
@@ -38,4 +38,10 @@ spec:
     component: agent
     release: {{ .Release.Name }}
     jaeger-infra: agent-instance
+{{- if .Values.agent.service.loadBalancerSourceRanges }}
```

Comment:
This block is repeated four times, exactly the same. Please, define a template and reuse it instead.